### PR TITLE
Add toast notification display

### PR DESCRIPTION
**Pull request by jeff-phillips-18**

### CODE REVIEW

File: app/views/landing-page.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <default-header class="top-header"></default-header>
+<toast-notifications></toast-notifications>
```
Comment (spadgett) [inline app/views/landing-page.html]: I spoke to @jwforres this morning, and we were both thinking we should just add this to index.html so it's available everywhere. I don't see a reason to wait to use toasts where it makes sense.

**Comment by spadgett** [comment]
[merge]

**Comment by spadgett** [comment]
@jeff-phillips-18 Can you try `hack/install-deps.sh && grunt build`? It's possible a dependency changed. Merge failed from a diff in vendor.js

**Comment by jeff-phillips-18** [comment]
@spadgett done, seems there was a change in the bootstrap-treeview dependency.

**Comment by spadgett** [comment]
@jeff-phillips-18 thanks

[merge]

**Pull request merged**